Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Economizer/Subsequences/Validation/Tuning_uEcoSta_uTowFanSpeMax.mo

```diff
@@ -0,0 +1,141 @@
+within Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Economizer.Subsequences.Validation;
+model Tuning_uEcoSta_uTowFanSpeMax
+  "Validate water side economizer tuning parameter sequence"
+
+  Tuning wseTun
```

Comment:
it would be better to use full path `Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Economizer.Subsequences.Tuning`?